spyder/widgets/browser.py

```diff
@@ -133,12 +133,12 @@ def setup(self, options={}):
             context=Qt.WidgetWithChildrenShortcut,
         )
 
-        copy_action = self.pageAction(QWebEnginePage.Copy)
+        original_copy_action = self.pageAction(QWebEnginePage.Copy)
         copy_action = self.create_action(
             name=WebViewActions.Copy,
             text=_("Copy"),
             # icon=self.create_icon(''),

```

Comment:
@goanpeca kind of unrelated but maybe could be worthy to remove the comment above?

Reply:
Fixed.